addons/mail/static/src/models/message/message.js

```diff
@@ -124,6 +124,9 @@ function factory(dependencies) {
                     this.env.models['mail.notification'].convertData(notificationData)
                 )]];
             }
+            if ('partner_ids' in data) {
+                data2.isCurrentPartnerMentioned = data.partner_ids.includes(this.env.messaging.currentPartner.id);
+            }
```

Comment:
Are you sure this is working? Isn't partner_ids the result of name get? with [id, display_name] instead of just list of id?

Reply:
I checked and it is working, this is really strange. `message_format` does not seem to be doing any post processing for `partner_ids`, I'm surprised `read` returns a list of ids for m2m instead of `name_get` result.